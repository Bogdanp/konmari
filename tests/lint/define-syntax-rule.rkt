#lang racket/base

(require (for-syntax racket/base))

(define-syntax-rule (define-keywords id ...)
  (begin
    (provide id ...)
    (define-syntax (id stx)
      (raise-syntax-error 'id "not allowed outside special form" stx)) ...))

(define-keywords foo bar baz)
