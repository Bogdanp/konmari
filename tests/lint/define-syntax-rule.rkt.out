/define-syntax-rule.rkt:11:18:warning:identifier 'foo' is never used
